updated readme

cg-cnu opened the pull request:
Hey @krathjen 
Just made some changes to readme. 
Organized the data a little and added some from the website. 
Feel free to suggest or make more changes. 
Hope it helps! Thanks 😄

krathjen commented:
Thanks @cg-cnu! I appreciate the help and the changes are looking good!

I'll suggest some other changes soon.

krathjen commented on README.md:
```diff
@@ -1,13 +1,21 @@
-## Studio Library
+<p align="center">
```
Would it be possible to have the icon to the left of the title and then align the title to the left?

krathjen commented on README.md:
```diff
@@ -1,13 +1,21 @@
+    />
+  </a>
+  <h1 align="center">Studio Library</h1>
+  <h4 align="center">Animation and pose manager for Maya</h4>
```
Let's remove "Animation and pose manager for Maya" and just have the description.

krathjen commented on README.md:
```diff
@@ -1,13 +1,21 @@
-Studio Library is a free python script for managing poses and animation in Maya.
 
-* www.studiolibrary.com
+## Description :
```
I don't think we need the "Description" header, so maybe just have "A Free tool ..." under the title without the header.

krathjen commented on README.md:
```diff
@@ -1,13 +1,21 @@
 
 
-### Features
+## Features :
```
Let's remove the ":" from all the subheaders. I think this looks a bit cleaner.

cg-cnu commented:
Hey @krathjen made the changes mentioned in the comments.
* Did a left alignment for log and text 
* Removed the tag line
* removed description header 
* removed `:` for more clarity

krathjen commented:
Thanks @cg-cnu! The update looks good!! I'll merge this with the develop branch for now.
